src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -167,36 +169,99 @@ function DeploymentConfigForm({ respondOnSuccess }) {
         }
     }
     const appMetricsEnvironmentVariableEnabled = window._env_ && window._env_.APPLICATION_METRICS_ENABLED;
+    let uniqueCharts = new Map<string, boolean>();
+    let chartNames = chartVersions.filter(cv => {if (uniqueCharts.get(cv.name)) { return false } else { uniqueCharts.set(cv.name, true);  return true;}});
```

Comment:
variable name assignment is irrelevant here.

Reply:
for now lets keep it